srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRiskModel.java

```diff
@@ -17,6 +33,146 @@
     
     private String modelName;
     
+    private List<Specialty> applicableSpecialties;
+    
+    private static final Logger fLogger = LoggerFactory.getLogger(EditRiskModel.class);
+
+    /**
+     * Class to hold the label, type and coefficient for a (link @ModelTerm) 
+     * and its coefficient on the Admin's Edit Model page. 
+     * 
+     * @author owner
+     *
+     */
+    public static class ModelTermSummary  implements Comparable<ModelTermSummary>
+    {
+        private final ModelTerm targetTerm;
+        
+        private String displayName;   // the name presented on the Edit Model page
+        private String termType;      // the variable type
```

Comment:
Ok. I'll take this as a general comment on all getters so no need for you to make a note of the others.